src/scripts/background/types.ts

```diff
@@ -25,19 +21,17 @@ export const DEFAULT_SKIP_OPTIONS: SkipOptions = {
   recap: 'manual-skip',
 } as const;
 
-export type SkipTimeIndicatorColours = Omit<
-  {
-    [T in SkipType]: string;
-  },
-  'preview'
->;
+export type SkipTimeIndicatorColours = {
```

Comment:
colours :?

Reply:
Its the type for the skip time indicator dictionary which maps a skip type to a colour